Formula/xcb-proto.rb

```diff
@@ -27,9 +24,9 @@ def install
       --disable-silent-rules
     ]
 
+    system "./autogen.sh"
```

Comment:
I guess you checked that it does not compile without it, right?

Reply:
If this line is necessary, could you add a comment describing why, preferably with the error message that it fixes?